proto/xudrpc.proto

```diff
@@ -356,14 +356,18 @@ message Peer {
   string address = 1 [json_name = "address"];
   // The node pub key to uniquely identify this peer
   string node_pub_key = 2 [json_name = "node_pub_key"];
+  // The LND BTC pub key associated with this peer
```

Comment:
I see "LND" usually capitalized as lower case "lnd" - including on the lnd main readme - so I'd stick with that.